docs/run-random-beacon.adoc

```diff
@@ -354,49 +383,25 @@ Keep uses the Ethereum Ropsten Testnet.
 
 ==== Contracts
 
+Contract addresses needed to boot the Random Beacon client.
+
 [%header,cols=2*]
 |===
 |Token
 |
 
-|KeepToken
-|`0xEF26cE3a729d4420558B10b8d9475CB84DA832c1`
-
 |TokenStaking
-|`0x585A83EdA3d29e030C49A51B46B653FEcd9c5919`
-
-|TokenGrant
-|`0xbf9c8f1125a34725CCe9a11a1b32509ED6773390`
-|===
-
-[%header,cols=2*]
-|===
-|Group
-|
-
-|Groups
-|`0xc75Df8B0aF13EFba5b44Fa2585998b208378c2E2`
-
-|GroupSelection
-|`0x49e75D5714fb92b061ED1863626BCBd458E2370f`
-|===
+|`0xE2Eda1d802e4aB47631500e3e6F30118350C5f1B`
```

Comment:
I think you deleted the table terminator here (`|===`), which seems to be causing an issue with rendering; either that or you meant to delete the header that follows.